src/bar.cpp

```diff
@@ -260,7 +260,7 @@ void waybar::Bar::onMap(GdkEventAny* ev) {
 
 void waybar::Bar::setExclusiveZone(uint32_t width, uint32_t height) {
   auto zone = 0;
-  if (visible) {
+  if (visible && config["layer"] != "overlay") {
```

Comment:
`sway-bar(5)`:
> (in overlay mode the bar is transparent to input events)

I wonder if it makes sense to support overlay without that part...

Also, if we're speaking about `bar mode overlay`, I assume that the layer should be 'top'. Need to check swaybar code to confirm though.
Otherwise we will see waybar on top of the lockscreen or fullscreen windows and that's not nice

Reply:
Yes, I removed this on the latest code. One thing at a time.

We can support only `mode "hide"` for now.